vertx-web/src/main/java/io/vertx/ext/web/impl/RouteImpl.java

```diff
@@ -295,35 +297,19 @@ synchronized boolean matches(RoutingContext context, String mountPoint, boolean
     }
     if (!consumes.isEmpty()) {
       // Can this route consume the specified content type
-      String contentType = request.headers().get("content-type");
-      boolean matches = false;
-      for (String ct: consumes) {
-        if (ctMatches(contentType, ct)) {
-          matches = true;
-          break;
-        }
-      }
-      if (!matches) {
-        return false;
-      }
+      // MYTODO How should this be? Cast or add to interface?
+      ParsedMIME contentType = ((HttpServerRequestWrapper) request).parsedHeaders().contentType;
```

Comment:
In that case, I'll have to send a PR to the core to change the interface or I have to make an interface for `vertx-web` that extends `vert.x-core`'s interface.
Which one is best in the context of this project?